label_studio/core/views.py

```diff
@@ -120,3 +122,13 @@ def samples_time_series(request):
     response['Content-Disposition'] = f'attachment; filename="{filename}"'
     response['filename'] = filename
     return response
+
+
+def localfiles_data(request):
+    """Serving files for LocalFilesImportStorage"""
+    path = request.GET.get('d')
+    local_serving_allowed = get_bool_env('LOCAL_FILES_SERVING_ENABLED', default=False)
+    if local_serving_allowed and path and request.user.is_authenticated:
+        return serve(request, path, document_root='/')
```

Comment:
I'd like to have here a more narrow path, maybe it's better to have it in env vars.

Reply:
Added LOCAL_FILES_DOCUMENT_ROOT variable to config document root value.